src/components/Hero.tsx

```diff
@@ -1,47 +1,60 @@
-import Image from 'next/image'
-import { urlForImage } from '~/lib/sanity.image'
-import { Button } from './Button'
-import Container from './Container'
-import { HeroType } from '~/lib/sanity.queries/hero/types'
+import Image from 'next/image';
+import { urlForImage } from '~/lib/sanity.image';
+import Navigation from '~/components/Navigation'
+import Container from './Container';
+import { HeroType } from '~/lib/sanity.queries/hero/types';
 
 const Hero = ({
   backgroundImage,
   description,
   title,
-  buttonName,
+  fontColor,
+  navbar,
+  logo
 }: HeroType) => {
-  const handleButtonClick = () => {
-    window.open(buttonName.redirectTo, '_blank')
-  }
+;
+  const renderRestOfHeader = (title) => {
+    const firstSpaceIndex = title.indexOf(' ');
+    return title.substring(firstSpaceIndex + 1);
+  };
+
+  const firstWord = title.split(' ')[0];
+  const restOfTitle = renderRestOfHeader(title);
 
   return (
-    <div className="relative bg-cover bg-center md:h-[40vh] flex flex-col justify-center items-center gap-4">
+    <>
+   
+    <div className='relative bg-cover bg-center'>
       {backgroundImage && (
-        <Image
-          className="absolute top-0 left-0 w-full h-full object-cover z-[-1]"
-          src={urlForImage(backgroundImage)?.url() || ''}
-          fill={true}
-          objectFit="cover"
-          alt="Logo UAWIT"
-        />
+        <div className=''>
+          <Image
+            className='absolute top-0 left-0 w-full h-full object-cover z-[-1]'
+            src={urlForImage(backgroundImage)?.url() || ''}
+            fill={true}
+            objectFit='cover'
+            alt='Logo UAWIT'
+          />
+          <div className='absolute inset-0 z-[-1] hero-overlay'></div>
+        </div>
       )}
-      <Container>
```

Comment:
I think you need a container here, all content should be in container for all blocks